### ISSUE TITLE
Create View or Export for a Mead Batch with Events (and Readings?)

### ISSUE THREAD

**Issue by Darkwater23** [Darkwater23/MeadMate#82]
It would be useful to see the entire batch lifecycle in one view and possible one extract. The current view shows initial properties of some fields and the latest event and reading. To paint a picture in chronological order, it would be useful to the starting data with the events and reading in order which should end with a bottled batch. Screenshot is easy enough, but a other formats for Reddit or web posting may be useful, as well.

**Comment by marc2982** [comment]
Yes this please!! A full view of the mead from yeast pitch to tasting notes would be super cool and helpful.

**Comment by TravisEvashkevich** [comment]
This would be an excellent feature +1 for want!
